echo-notifications/src/test/groovy/com/netflix/spinnaker/echo/notification/DryRunNotificationAgentSpec.groovy

```diff
@@ -71,7 +71,7 @@ class DryRunNotificationAgentSpec extends Specification {
 
   def "triggers a pipeline run for a pipeline:complete notification"() {
     given:
-    front50.getPipelines(application) >> Observable.just([pipeline])
+    front50.getPipelines(application) >> [pipeline]
```

Comment:
Can now remove the import of `rx.Observable` above.  That's also the last direct reference to `rxJava` in `echo-notifications` so it can be removed from the `echo-notifications.gradle` file.

Reply:
Sweet llamas of the Bahamas!